### DISCUSSION TITLE
Disable inspect mode

### DISCUSSION THREAD

**Discussion by karanmidha** [pocka/figspec#48] [Q&A]
Is there a way I can disable the inspect mode for `<FigSpecFileViewer />`?

@pocka @yannbf

**Comment by pocka** [comment]
No.

Adding a cancellable `inspectoropen` event and/or `part` attribute to the inspector panel would be good candidate, but currently no plan to do as I don't think this library without inspector panel is any useful compared to Figma Live Embed Kit. Please open an issue with `enchancement` label if you have a concrete usecase.
